WNPRC_Purchasing/resources/queries/ehr_purchasing/purchasingRequestsOverviewForAdmins.sql

```diff
@@ -0,0 +1,19 @@
+SELECT
+       pr.rowId AS requestNum,
+       pr.vendorId,
+       pr.account,
+       pr.otherAcctAndInves,
+       pr.qcState   AS requestStatus,
+       pr.created   AS requestDate,
+       pr.createdBy AS requester,
+       items.totalCost,
+       pr.assignedTo
+FROM ehr_purchasing.purchasingRequests pr
+LEFT JOIN
+    (
+        SELECT requestRowId,
+               round(sum(quantity * unitCost), 2) AS totalCost
```

Comment:
There are a number of ROUND() calls that we should be able to remove after changing the unitCost data type

Reply:
done